sds_data_manager/lambda_code/SDSCode/dependency_config.csv

```diff
@@ -67,10 +67,27 @@ hit, l1a, hk, hit, l1b, hk, HARD, DOWNSTREAM
 idex, l0, raw, idex, l1, sci, HARD, DOWNSTREAM
 
 # <---- LO Dependencies ---->
-# TODO: add LO dependencies
-lo, l0, raw, lo, l1a, all, HARD, DOWNSTREAM
-
-lo, l1a, histogram, lo, l1b, histogram, HARD, DOWNSTREAM
+lo, l0, de, lo, l1a, de, HARD, DOWNSTREAM
+lo, l0, histogram, lo, l1a, histogram, HARD, DOWNSTREAM
+lo, l0, star, lo, l1a, star, HARD, DOWNSTREAM
+lo, l0, spin, lo, l1a, spin, HARD, DOWNSTREAM
+lo, l0, hk, lo, l1a, hk, HARD, DOWNSTREAM
+
+lo, l1a, de, lo, l1b, de, HARD, DOWNSTREAM
+lo, l1a, spin, lo, l1b, de, HARD, DOWNSTREAM
+lo, l1a, tbd, lo, l1b, badtimes, HARD, DOWNSTREAM
```

Comment:
It might be good to make this all caps `TBD`, or a different word `TODO` and something that will stick out more to us in the editors? Otherwise it looks like `tbd` might be a valid name at first glance here.
```suggestion
lo, l1a, TODO, lo, l1b, badtimes, HARD, DOWNSTREAM
```